### ISSUE TITLE
vmtkmeshgenerator problem

### ISSUE THREAD

**Issue by Lorenzomarta** [pyvista/tetgen#37]
Hello everyone,
I’m having trouble using the command “vmtkmeshgenerator” to obtain internal tetrahedral mesh, in particular starting with this command:

vmtk vmtkmeshgenerator -ifile sup3.vtp -skipremeshing 1 -volumeelementfactor 1 -ofile mesh.vtu

I obtain the following output:

[...]
Duplicating background mesh.
Internalerror in findedge():  Unable to find an edge in subface.
ERROR: In ../vtkVmtk/Misc/vtkvmtkTetGenWrapper.cxx, line 431
vtkvmtkTetGenWrapper (0x7fa3b27ef170): TetGen quit with an exception.
[...]

What do you suggest me to do?
Thanks in advance.

**Comment by banesullivan** [comment]
I don't think this is the right place for your issue. This python wrapping of TetGen has nothing to do `vtkvmtkTetGenWrapper`.

I suggest you open an issue in vmtk: https://github.com/vmtk/vmtk

**Comment by banesullivan** [comment]
Seems https://github.com/vmtk/vmtk/issues/375 is tracking this or is at least related.
